Block/Form.php

```diff
@@ -0,0 +1,186 @@
+        }
+
+        $result = [
+            'customerAddressId' => $billingAddress->getId(),
+            'countryId'         => $billingAddress->getCountryId(),
+            'regionId'          => $billingAddress->getRegionId(),
+            'regionCode'        => $billingAddress->getRegionCode(),
+            'region'            => $billingAddress->getRegion(),
+            'customerId'        => $billingAddress->getCustomerId(),
+            'street'            => $streetAddressLines,
+            'telephone'         => $billingAddress->getTelephone(),
+            'postcode'          => $billingAddress->getPostcode(),
+            'city'              => $billingAddress->getCity(),
+            'firstname'     => $billingAddress->getFirstname(),
+            'lastname'      => $billingAddress->getLastname(),
+            'extensionAttributes'   => ['checkoutFields' => []],
+            'saveInAddressBook'     => null
+        ];
+
+        return ($needJsonEncode) ? json_encode($result) : $result;
+    }
+
+    public function getPlaceOrderPayload()
+    {
+        $quote = $this->getQuoteData();
+
+        $result = [
+            'cartId' => $quote->getId(),
+            'billingAddress' => $this->getBillingAddress(false),
+            'paymentMethod' => [
+                'method' => $quote->getPayment()->getMethod(),
+                'po_number' => null,
+                'additional_data' => null
+            ],
+            'email' => $quote->getCustomerEmail()
+        ];
+
+        return json_encode($result);
+    }
+
+    /**
+     * @return string
+     */
+    public function getJavascriptSuccess()
+    {
+        return $this->configHelper->getJavascriptSuccess();
+    }
+
+    /**
+     * Get Replace Button Selectors.
+     *
+     * @return string
+     */
+    public function getGlobalCSS()
+    {
+        return $this->configHelper->getGlobalCSS();
+    }
+
+    /**
+     * Get Javascript page settings.
+     *
+     * @return string
+     */
+    public function getSettings()
+    {
+        return json_encode([
+            'connect_url'              => $this->getConnectJsUrl(),
+            'publishable_key_payment'  => $this->getCheckoutKey(),
+            'create_order_url'         => $this->getUrl(Config::CREATE_ORDER_ACTION),
+            'save_order_url'           => $this->getUrl(Config::SAVE_ORDER_ACTION),
+            'selectors'                => $this->getReplaceSelectors(),
+        ]);
+    }
+
+    /**
+     * @return string
+     */
+    public function getConnectJsUrl()
+    {
+        // Get cdn url
+        $cdnUrl = $this->configHelper->getCdnUrl();
+
+        return $cdnUrl . '/connect.js';
+    }
+
+    /**
+     * Get checkout key. Any of the defined publishable keys for use with track.js.
+     *
+     * @return  string
+     */
+    public function getCheckoutKey()
+    {
+        return $this->configHelper->getPublishableKeyPayment();
```

Comment:
We need to add a new config named `publishable_key_back_office` in `system.xml`. 
We need separate publishable key for back office payment processing because fee is different.